metric-schema-java/src/main/java/com/palantir/metric/schema/JavaGenerator.java

```diff
@@ -18,16 +18,21 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.errorprone.annotations.CanIgnoreReturnValue;
+import com.palantir.goethe.Goethe;
 import java.nio.file.Path;
 import java.util.List;
 import java.util.Optional;
 
 public final class JavaGenerator {
     // TODO(forozco): consider splitting this out into an API package
-    /** Specifies under which package Java classes should be generated. */
+    /**
+     * Specifies under which package Java classes should be generated.
+     */
     private static final String JAVA_PACKAGE = "javaPackage";
 
-    /** Specifies visibility of generated Utility class. Defaults to public */
+    /**
+     * Specifies visibility of generated Utility class. Defaults to public
+     */
```

Comment:
I wish we could retain the nice one-line javadoc with pjf, but that's a nit for another day :-)